### PR TITLE
Fix grammar and consistency in README

### PR DESCRIPTION
**Pull request by LBBO**
Refer to AsciiDoc and AsciiDoctor.js more consistently

Fix grammar mistakes and add new ones

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -1,20 +1,18 @@
...
-We just thought it would be even greater with asciidoctor.js ;)
-
-So we integrated https://github.com/asciidoctor/asciidoctor.js[asciidoctor.js] and changed the name accordingly.
+We just thought it would be even greater with AsciiDoctor and so we included https://github.com/asciidoctor/asciidoctor.js[Asciidoctor.js] into the project and changed its name accordingly ;)
 
 == Status
 
-Using AsciiDoc.js, the base functionality, typing AsciiDoc and seeing a nice rendered document is working fine.
-Also, uploading and embedding images is working.
+Thanks to Asciidoc.js, the base functionality (typing AsciiDoc and seeing a nice rendered document) workis fine.
```
Comment (fliiiix) [inline README.adoc]: > the base functionality (typing AsciiDoc and seeing a nice rendered document) **workis fine.**

typo?

**Review by fliiiix — CHANGES_REQUESTED** [review]

**Comment by fliiiix** [comment]
The rest looks fine to me :+1: Thanks :heart:

File: README.adoc (hunk 1 above)
Comment (LBBO) [reply README.adoc]: Yepp :grin: Fixed it with 759249ef452fded76975371daf74659864e9ff93

**Review by fliiiix — APPROVED** [review]

**Comment by fliiiix** [comment]
good job

**Pull request merged**